test/ngSanitize/filter/linkySpec.js

```diff
@@ -54,4 +54,19 @@ describe('linky', function() {
     expect(linky("http://example.com", "someNamedIFrame")).
       toEqual('<a target="someNamedIFrame" href="http://example.com">http://example.com</a>');
   });
+
+  it('should optionally add custom attributes', function() {
+    expect(linky("http://example.com", "_self", {rel: "nofollow"})).
+      toEqual('<a rel="nofollow" target="_self" href="http://example.com">http://example.com</a>');
+  });
+
+  it('should optionally add custom attributes from function', function() {
```

Comment:
There should also be an exra test that ensures the url is actually passed to the attr function